GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -465,17 +465,12 @@ public void DisableContextMenu()
         internal void SuspendRefreshRevisions() => _updatingFilters++;
 
         /// <summary>
-        ///  Resume revisions refreshes and invokes <see cref="ForceRefreshRevisions"/>.
+        ///  Resume revisions refreshes.
         /// </summary>
         internal void ResumeRefreshRevisions()
         {
             --_updatingFilters;
             Debug.Assert(_updatingFilters >= 0, $"{nameof(ResumeRefreshRevisions)} was called without matching {nameof(SuspendRefreshRevisions)}!");
-
-            if (_updatingFilters == 0)
-            {
-                ForceRefreshRevisions();
-            }
```

Comment:
Coupled with above comment I'm finding this change the most puzzling. This was meant to guard against nested unbalanced suspensions (i.e., without a resume call), and execute a refresh only on the outer scope.
Now it feels like we may be returning into a realm where we may be calling multiple refreshes.

Reply:
See above, the refresh is always called explicitly after Suspend/Resume.